src/window.rs

```diff
@@ -947,6 +947,9 @@ impl Window {
     ///
     /// ## Platform-specific
     ///
+    /// - **Wayland:** You can also use `WINIT_WAYLAND_CSD_THEME` env variable to set the theme.
+    /// Possible values for env variable are: "dark" and light". When unspecified, a theme is automatically selected.
+    ///
     /// - **iOS / Android / Web / x11:** Unsupported.
```

Comment:
Should update x11 docs here as well. Probably if you've decided to touch gtk theme you should also update builder.